src/EditorFeatures/Core.Wpf/Suggestions/AsyncSuggestedActionsSource.cs

```diff
@@ -118,7 +119,7 @@ private partial class AsyncSuggestedActionsSource : SuggestedActionsSource, IAsy
                                 state, requestedActionCategories, document,
                                 range, selection,
                                 addOperationScope: _ => null,
-                                includeSuppressionFixes: priority.Value == CodeActionRequestPriority.Normal,
+                                includeSuppressionFixes: priority.Value == CodeActionRequestPriority.Low,
                                 priority.Value,
```

Comment:
interesting.  we do pass priority in here.  i think includeSuppression is likely redundant.

Reply:
Yes, removed now.